packages/cli-vector/package.json

```diff
@@ -49,5 +53,16 @@
   "files": [
     "build/",
     "dist/"
-  ]
+  ],
+  "dependencies": {
+    "@linzjs/docker-command": "^7.5.0",
+    "@mapbox/geojson-area": "^0.2.2",
```

Comment:
We have `Area` in @basemaps/geo does this need to exist?

Reply:
I think this is used to calculate the size of polygon for large lakes, do we have the same function in `Aera`.

Reply:
> I think this is used to calculate the size of polygon for large lakes, do we have the same function in Aera.

Area can give it to you in sqkm, if you reproject it to 2193, otherwise you can run it on the latlngs and it will give you it in square degrees which can be used to approximate the size of the lakes